cmd/podman/oci-trace-hook/trace.go

```diff
@@ -0,0 +1,238 @@
+package main
```

Comment:
The hook should be placed under `cmd/oci-trace-hook/`. I don't see why this should be under `cmd/podman`.